src/libraries/System.Runtime.Intrinsics/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
Why do we have all these compatibility suppressions instead of adding these API to ref assemblies?